binrz/rz-test/rz-test.c

```diff
@@ -59,6 +59,7 @@ static void interact(RzTestState *state);
 static bool interact_fix(RzTestResultInfo *result, RzPVector /*<RzTestResultInfo *>*/ *fixup_results);
 static void interact_break(RzTestResultInfo *result, RzPVector /*<RzTestResultInfo *>*/ *fixup_results);
 static void interact_commands(RzTestResultInfo *result, RzPVector /*<RzTestResultInfo *>*/ *fixup_results);
+static void accept_all(RzTestState *state);
 
 static int help(bool verbose) {
```

Comment:
> https://github.com/rizinorg/rizin/commit/042f9174a4b5568975af7a4d387736c7f3452ab5

> 
> For example, in this commit, I used this feature to automatically ACCEPT over 200 asm tests at a time.

Ok. Please add the flag to `rz-test -h`.

Reply:
@imbillow could you please address this request, then we can merge?